src/main/java/org/fogbowcloud/saps/engine/core/scheduler/executor/JobExecutionService.java

```diff
@@ -0,0 +1,17 @@
+package org.fogbowcloud.saps.engine.core.scheduler.executor;
+
+import java.util.List;
+import org.fogbowcloud.saps.engine.core.scheduler.executor.arrebol.dtos.JobResponseDTO;
+import org.fogbowcloud.saps.engine.core.model.SapsJob;
+
+public interface JobExecutionService {
```

Comment:
Wouldn't it be better to create an exception for this interface and use it in methods?